_includes/head.html

```diff
@@ -0,0 +1,74 @@
+      [ng\:cloak],
+      [ng-cloak],
+      [data-ng-cloak],
+      [x-ng-cloak],
+      .ng-cloak,
+      .x-ng-cloak,
+      .ng-hide:not(.ng-hide-animate) {
+          display: none !important;
+      }
+
+      ng\:form {
+          display: block;
+      }
+  </style>
+  <!-- Google Tag Manager -->
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer','GTM-WL2QLG5');</script>
+  <!-- End Google Tag Manager -->
+  {% if site.GH_ENV == "gh_pages" %}
+  <meta name="robots" content="noindex">{% endif %}
+  <!-- favicon -->
+  <link rel="icon" type="image/x-icon" href="/favicons/docs@2x.ico" sizes="129x128">
+  <meta name="msapplication-TileImage" content="/favicons/docs@2x.ico">
+  <link rel="apple-touch-icon" type="image/x-icon" href="/favicons/docs@2x.ico" sizes="129x128">
+  <meta property="og:image" content="/favicons/docs@2x.ico"/>
+  <!-- metadata -->
+  <meta property="og:type" content="website"/>
+  <meta property="og:updated_time" itemprop="dateUpdated" content="{% if page.date %}{{ page.date | date_to_xmlschema }}{% else %}{{ site.time | date_to_xmlschema }}{% endif %}"/>
+  <meta property="og:image" itemprop="image primaryImageOfPage" content="/images/docs@2x.png"/>
+  <meta name="twitter:card" content="summary"/>
+  <meta name="twitter:domain" content="docs.docker.com"/>
+  <meta name="twitter:site" content="@docker_docs"/>
+  <meta name="twitter:url" content="https://twitter.com/docker_docs"/>
+  <meta name="twitter:title" itemprop="title name" content="{{ page.title }}"/>
+  <meta name="twitter:description" property="og:description" itemprop="description" content="{{ content | strip_html | truncatewords: 30}}" />
+  <meta name="twitter:image:src" content="/images/docs@2x.png"/>
+  <meta name="twitter:image:alt" content="Docker Documentation"/>
+  <meta property="article:published_time" itemprop="datePublished" content="{% if page.date %}{{ page.date | date_to_xmlschema }}{% else %}{{ site.time | date_to_xmlschema }}{% endif %}"/>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="keywords" content="{% if page.keywords %}{{ page.keywords }}{% else %}docker, docker open source, docker platform, distributed applications, microservices, containers, docker containers, docker software, docker virtualization{% endif %}">
+  <link rel="stylesheet" href="/css/font-awesome.min.css">
+  <link rel="stylesheet" href="/css/bootstrap.min.css">
+  <link id="pygments" rel="stylesheet" href="/css/pygments/perldoc.css">
+  <link id="pagestyle" rel="stylesheet" href="/css/style.css">
+
+  <!-- Go get "Open Sans" font from Google -->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```

Comment:
Something to look into; we're fetching these styles and fonts directly from fonts.googleapis.com. Not sure if things improved, but my experience from way-back-when was that this was _slow_ (and I think they disabled caching so that metrics could be collected). Perhaps no longer the case, but if it is, we should look at hosting those fonts and styles locally (like we do for the other font)